src/store/index.js

```diff
@@ -3,6 +3,7 @@
 //
 //  Created by Kalila L. on 2 Jul 2020.
 //  Copyright 2020 Vircadia contributors.
+//  Modified for Overte. 2022 Overte e.V.
```

Comment:
The way we have always handled this is to add our copyrigth:
```suggestion
//  Copyright 2022 Overte e.V.
```

Reply:
Done